src/libmongoc/src/mongoc/mongoc-apm-private.h

```diff
@@ -188,11 +193,25 @@ mongoc_apm_command_failed_init (mongoc_apm_command_failed_t *event,
                                 int64_t operation_id,
                                 const mongoc_host_list_t *host,
                                 uint32_t server_id,
+                                bool force_redaction,
                                 void *context);
 
 void
 mongoc_apm_command_failed_cleanup (mongoc_apm_command_failed_t *event);
 
+bool
+mongoc_apm_is_sensitive_command (const char *command_name,
+                                 const bson_t *command);
+
+void
+mongoc_apm_redact_command (bson_t *command);
```

Comment:
I suggest moving the declarations of `mongoc_apm_redact_command` and `mongoc_apm_redact_reply` to the top of mongoc-apm.c and marking those static.

That makes it clearer to readers that these are helpers only needed in mongoc-apm.c, and not intended to be called elsewhere.